src/main.ts

```diff
@@ -76,30 +76,33 @@ async function getDockleEnvVariables(): Promise<{ [key: string]: string }> {
 
 async function runTrivy(): Promise<number> {
     const trivyPath = await trivyHelper.getTrivy();
-    console.log(util.format("Trivy executable found at path ", trivyPath));
+    core.debug(util.format("Trivy executable found at path ", trivyPath));
     const trivyEnv = await getTrivyEnvVariables();
 
     const imageName = inputHelper.imageName;
     const trivyOptions: ExecOptions = {
         env: trivyEnv,
-        ignoreReturnCode: true
+        ignoreReturnCode: true,
+        silent: true
     };
-
+    console.log("Scanning for vulnerabilties...");
     const trivyToolRunner = new ToolRunner(trivyPath, [imageName], trivyOptions);
     const trivyStatus = await trivyToolRunner.exec();
     return trivyStatus;
 }
 
 async function runDockle(): Promise<number> {
     const docklePath = await dockleHelper.getDockle();
+    core.debug(util.format("Dockle executable found at path ", docklePath));
     const dockleEnv = await getDockleEnvVariables();
     const imageName = inputHelper.imageName;
 
     const dockleOptions: ExecOptions = {
         env: dockleEnv,
-        ignoreReturnCode: true
+        ignoreReturnCode: true,
+        silent: true
```

Comment:
You'll have to add log parsing logic to find failures of scan too.

Reply:
I think we can add those changes in a separate PR as it needs some more investigation. Will be tracking the issue [here](https://github.com/shigupt202/local-container-scan/issues/11).